_includes/documentation-links-band.html

```diff
@@ -91,7 +91,7 @@ <h4><i class="fas fa-user-friends"></i> Community</h4>
           <a href="https://how-to.vertx.io/web-session-infinispan-howto/" target="_blank">Vert.x Web Sessions</a>
         </li>
         <li>
-          <a href="https://www.keycloak.org/docs/latest/server_installation" target="_blank">Keycloak</a>
+          <a href="https://www.keycloak.org/high-availability/introduction " target="_blank">Keycloak</a>
```

Comment:
We maybe want to link to this instead as it's applicable to more deployment types. HA is pretty niche still.

https://www.keycloak.org/server/caching